class.c

```diff
@@ -1589,6 +1589,33 @@ rb_class_subclasses(VALUE klass)
     return class_descendants(klass, true);
 }
 
+/*
+ *  call-seq:
+ *     subclasses -> object
```

Comment:
I think you forgot to update this part of the docs:

```suggestion
 *     attached_object -> object
```